### ISSUE TITLE
Add KC_LOCK support by adding KEY_LOCK_ENABLE = yes to rules.mk

### ISSUE THREAD

**Issue by JonasWechsler** [qmk/qmk_configurator#236]
I'd like to be able to map KC_LOCK with QMK Configurator. I can add the key with the "any" key, but it doesn't set the appropriate flag in rules.mk, so it doesn't compile.

**Comment by yanfali** [comment]
Yup, sorry we don't support this quite yet. API support for setting flags is a work in progress.

**Comment by eraucher** [comment]
Using mousekeys for boards with "MOUSEKEY_ENABLE = no" in the board's default rules.mk file is also not possible with QMK Configurator, which really hinders its otherwise awesome functionality with certain boards. I'm curious if there are any hacky ways of getting around this.
